micro-rdk-ffi/build.rs

```diff
@@ -113,6 +113,6 @@ fn main() {
             .with_config(cfg)
             .generate()
             .expect("Unable to generate bindings")
-            .write_to_file("include/micrordk.h");
```

Comment:
why change this?

Reply:
oh I forgot to undo this change, sorry. I needed it temporarily for my local idf-component example setup to work. But I assume that problem will be fixed when your other PR moving the component to this repo goes in.